webpack.config.js

```diff
@@ -385,59 +416,24 @@ module.exports = async (env, argv) => {
           test: /\.html$/,
           loader: "html-loader",
           options: {
-            // <a-asset-item>'s src property is overwritten with the correct transformed asset url.
-            attrs: ["img:src", "a-asset-item:src", "audio:src", "source:src"]
-          }
-        },
-        {
-          test: /\.worker\.js$/,
-          loader: "worker-loader",
-          options: {
-            name: "assets/js/[name]-[hash].js",
-            publicPath: "/",
-            inline: true
+            minimize: false, // This is handled by HTMLWebpackPlugin
+            sources: {
+              list: [
+                { tag: "img", attribute: "src", type: "src" },
+                { tag: "a-asset-item", attribute: "src", type: "src" },
+                { tag: "audio", attribute: "src", type: "src" },
+                { tag: "source", attribute: "src", type: "src" }
+              ]
+            }
           }
         },
+        // Some JS assets are loaded at runtime and should be copied unmodified and loaded using file-loader
         {
-          test: [
-            path.resolve(__dirname, "src", "utils", "configs.js"),
-            path.resolve(__dirname, "src", "utils", "i18n.js"),
-            path.resolve(__dirname, "src", "support.js")
-          ],
-          loader: "babel-loader",
-          options: legacyBabelConfig
-        },
```

Comment:
I think it's reasonable to re-evaluate whether we need this legacy support. Ideally we should only have to care about evergreen browsers at this point. As a lower priority, we'll probably still want to do spot checks on how things behave in old-but-still-popular browsers. Mainly we want to avoid the situation where nothing is rendered at all and you just see a blank page in unsupported browsers. 
Lastly, in addition to old browsers, there are still edge cases like in-app-browsers that are used in mobile apps like Twitter and Facebook, where we'll want to check that things don't break entirely.

Reply:
From what I could tell, the way wee were polyfilling stuff we were only ever loading polyfills in hub.js anyway. Though I guess this could emit code with features that might not be supported. I think if the support page at least runs in IE 11 we can keep this out... How to test that is the question...

Reply:
You can use browserstack to do a quick check, but it would be easiest if it were deployed to a public server like dev.ret or smoke. Browserstack has the ability to test local servers, but hubs might be too complicated for that to work.

Reply:
Tested on IE11 and page failed to load on a syntax error for `=>`.. I think I misunderstood the intent of this block before. Having a restricted legacy babel config makes sense, we had just started using it for other things. I have restored this but inlined it, overriding only the target and keeping all plugins (so we don't accidentally make some code indectly linked to the support bundle that breaks it. 

In the future I think it would be best to just hand craft support.js to be functional without polyfilling. We don't need anything fancy in there, not even React.